cupy/cuda/function.pyx

```diff
@@ -84,6 +85,9 @@ cdef inline CPointer _pointer(x):
     if isinstance(x, CPointer):
         return x
 
+    if isinstance(x, TextureObject):
+        return CPointer(TextureObject.ptr)
```

Comment:
@emcastillo the bug is here...It should have been `x.ptr` lol

Reply:
hahah, so glad you found it! that was a tricky one.

Reply:
Thanks for helping me test this stupid mistake...